### PR TITLE
Sanity Updates - 2023-05-22

### PR DESCRIPTION
**Pull request by wrp801**
Updated Blueprint Library files with latest data from Sanity. 

 Run for *[_type == "vendors" && name match("Thoughtspot") && !(_id in path("drafts.**"))]

### CODE REVIEW

**Comment by wrp801** [comment]
This is ready to go now

File: docs/blueprint-library/thoughtspot/thoughtspot-authorization.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+id: Thoughtspot-authorization
+sidebar_position: 1
+title: Thoughtspot Blueprint Authorization
+hide_title: true
+sidebar_label: Authorization
+description: Instructions on how to authorize Coalesce to work with Shipyard's low-code Thoughtspot templates.
+keywords:
+  - Thoughtspot
```
Comment (johnathan-rodriguez) [inline docs/blueprint-library/thoughtspot/thoughtspot-authorization.md:9]: does the capitalization matter?

**Review by johnathan-rodriguez — APPROVED** [review]
If the capitalization does not matter for the id and keyword section this is approved

File: docs/blueprint-library/thoughtspot/thoughtspot-authorization.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+id: Thoughtspot-authorization
```
Comment (johnathan-rodriguez) [inline docs/blueprint-library/thoughtspot/thoughtspot-authorization.md:2]: same capitalization question here

**Comment by wrp801** [comment]
I made those lowercase to match the others

**Comment by johnathan-rodriguez** [comment]
I am assuming those details aren't customer facing, correct? @wrp801

**Comment by wrp801** [comment]
Yeah it’s just a part of the framework

**Review by johnathan-rodriguez — APPROVED** [review]

**Pull request merged**